### ISSUE TITLE
is_full() panics with Bitmap of size 32 

### ISSUE THREAD

**Issue by kumargu** [bodil/bitmaps#19]
checking for is_full with Bitmap of size 32 and all bits set to true, always panics. 

```
 let mut bitmap = Bitmap::<32>::mask(32);

    let z = bitmap.is_full();
    println!("Yes full {:?}", z);
```

```
thread 'main' panicked at 'attempt to shift left with overflow', /Users/kumargu/.cargo/registry/src/github.com-1ecc6299db9ec823/bitmaps-3.2.0/src/types.rs:550:1
```

**Comment by fadeevab** [comment]
@kumargu @bodil  I created a PR to fix this issue: #27. Thanks!

**Comment by bodil** [comment]
Fixed by #27.
